src/Umbraco.Infrastructure/Examine/IndexRebuilder.cs

```diff
@@ -14,12 +15,14 @@ namespace Umbraco.Examine
     /// </summary>
     public class IndexRebuilder
     {
-        private readonly IProfilingLogger _logger;
+        private readonly IProfilingLogger _pLogger;
```

Comment:
Just noticed that most of this time this variable is called `_profilingLogger` (or `profilingLogger` in the constructor), which is a little more readable/explanatory and easier to see which one is being used.  So small thing, but maybe could do a find/replace to standardise?